src/analysis/cytoscape/cytoscape.py

```diff
@@ -0,0 +1,50 @@
+from pathlib import PurePath
+from typing import List, Union
+from pathlib import PurePath, Path
+import os
+import docker
+
+def run_cytoscape_container(pathways: List[Union[str, PurePath]], out_dir: str) -> None:
```

Comment:
@ajshedivy and I discussed that we will first get this working reliably with Docker.  Then we'll work on generalizing it to use the container running utility to see if it can run in Singularity as well.